From 66293216bfbc428e94fec7241f84c441f4a0bcd4 Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Fri, 4 Apr 2008 17:05:31 -0500
Subject: [PATCH] ENGR00071507 Use the latest SDMA mcu_2_app script for MX31

Update the devices file to point to the latest mcu_2_app SDMA script

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 arch/arm/mach-mx3/devices.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx3/devices.c b/arch/arm/mach-mx3/devices.c
index 7ecec84..f2c727f 100644
--- a/arch/arm/mach-mx3/devices.c
+++ b/arch/arm/mach-mx3/devices.c
@@ -84,7 +84,8 @@ void mxc_sdma_get_script_info(sdma_script_start_addrs * sdma_script_addr)
 		sdma_script_addr->mxc_sdma_ap_2_bp_addr = ap_2_bp_ADDR_2;
 		sdma_script_addr->mxc_sdma_bp_2_ap_addr = bp_2_ap_ADDR_2;
 		sdma_script_addr->mxc_sdma_loopback_on_dsp_side_addr = -1;
-		sdma_script_addr->mxc_sdma_mcu_2_app_addr = mcu_2_app_ADDR_2;
+		sdma_script_addr->mxc_sdma_mcu_2_app_addr =
+		    mcu_2_app_patched_ADDR_2;
 		sdma_script_addr->mxc_sdma_mcu_2_shp_addr =
 		    mcu_2_shp_patched_ADDR_2;
 		sdma_script_addr->mxc_sdma_mcu_interrupt_only_addr = -1;
-- 
1.5.4.4

